src/patternfly/components/Title/docs/code.md

```diff
@@ -0,0 +1,14 @@
+## Overview
+The title component applies top and bottom margins, font-weight, font-size, and line-height to titles.
+## Usage
+
+| Class | Applied | Outcome
+| -- | -- | -- |
+| `.pf-c-title` | `<h1>`,`<h2>`,`<h3>`,`<h4>`,`<h5>`,`<h6>` |  Initiates a title. Always use it with a modifier class. |
+| `.pf-is-xxxxl` | `.pf-c-title` | Modifies for xxxxl title styles |
```

Comment:
Sorry I missed this before:
> Modifies for xxxxl title styles
means nothing to me. What it does is that it changes the font size to be xxxxl